symplyphysics/symbols/classical_mechanics.py

```diff
@@ -0,0 +1,96 @@
+"""
+Classical mechanics
+===================
+
+Symbols related to classical mechanics.
+"""
+
+from symplyphysics import units, SymbolNew, angle_type, clone_symbol, dimensionless
+
+force = SymbolNew("F", units.force)
+"""
+**Force** is a vector quantity denoting an influence that can cause an object to change its velocity
+unless counterbalanced by other forces.
+"""
+
+speed = SymbolNew("v", units.velocity)
+"""
+**Speed** is the rate of change of the object's position with respect to time. It is the scalar counterpart
+of the velocity vector, which is the rate of change of the object's displacement with respect to time.
+"""
+
+acceleration = SymbolNew("a", units.acceleration)
+"""
+**Acceleration** is the rate of change of the object's velocity with respect to time. It is a vector quantity.
+"""
+
+distance = SymbolNew("d", units.length)
+"""
+**Distance** is is a measure of the spatial separation between two points.
+"""
+
+radial_distance = SymbolNew("r", units.length)
+"""
+Distance to the origin of the coordinate system.
+"""
+
+distance_to_axis = SymbolNew("r", units.length)
```

Comment:
These two are pretty much the same as radius of curvature. Do we need them?

Reply:
Operationally they're different. The distance to axis is just the radial distance in cylindrical coordinates and does not imply any motion. Radius of curvature is related to the trajectiory of an object.

Reply:
Let's equate such symbols instead of cloning: `distance_to_axis = radius_of_curvature`

Reply:
What if one wants to use both of them in one law? For this reason I believe all symbols here should be distinct.

My point is that if two quantities are operationally different, they should be described by different symbols. So, for instance, distance (magnitude of the displacement vector), length (a dimension), distance to origin, distance to axis, radius of curvature, optical path length, proper length, etc all require distinct symbols because they describe different measurements.

Reply:
It seems to be quite a tricky problem. Let's keep it the way you did. Maybe eventually we'll find a way to consistently define global symbols.